cwms-data-api/src/main/java/cwms/cda/data/dto/TimeSeries.java

```diff
@@ -79,7 +79,8 @@ public class TimeSeries extends CwmsDTOPaginated {
 
     @ArraySchema(
             schema = @Schema(
-                    description = "List of retrieved time-series values",
+                    description = "List of retrieved time-series values. Contains [dateTime, value, qualityCode]. "
```

Comment:
The purpose for the description is to allow the array to be unspecified, the description should not identify what the columns are, rather the value-columns property is what tells the client what the columns are. ex, if we add a boolean "include-quality" in the future, we want to allow the array to be of size 2.

Reply:
I removed the column data description and added a note that the value-columns describe the value array data structure.